content/en/monitors/notifications.md

```diff
@@ -53,7 +53,19 @@ Add tags to your monitor (optional). Monitor tags are different than metric tags
 
 ### Renotify
 
-Enable monitor renotification (optional), which is useful to remind your team a problem is not solved. If enabled, you are given the option to include an escalation message sent any time the monitor renotifies. The original notification message is also included.
+Enable monitor renotification (optional), which is useful to remind your team a problem is not solved. 
+If enabled, you are given the option to include an escalation message sent any time the monitor remain in the `alert` or `no data` state for the configured time.
+The escalation message can be added in two ways:
+
+* In the escalation_message box (UI) or attribute (API)
+* (recommended) By leveraging the `{{#is_renotify}}` block in the original message
+
+The original notification message is always included therefore you want to leverage the `{{#is_renotify}}` block in the following conditions:
+
+1. The escalation message is only there to add some extra details and should not contain the original message details
+2. The escalation message should only be sent for a subset of groups
```

Comment:
```suggestion
1. Include only extra details in the `{{#is_renotify}}` block and don't repeat the original message details.
2. Send the escalation message to a subset of groups.
```
Is "to a subset of groups" correct? I wasn't sure what sending a message "for" a subset meant, but maybe it was correct.